scripts/neighbor_advertiser

```diff
@@ -397,25 +413,29 @@ def add_mirror_session(dst_ipv4_addr):
     config_db.set_entry('MIRROR_SESSION', MIRROR_SESSION_NAME, session_info)
 
 
-
 def add_mirror_acl_rule():
+    (v4_table, v6_table) = find_mirror_table_name()
+
     acl_rule_info = {
         'PRIORITY': '8888',
         'ether_type': '2054',
         'mirror_action': MIRROR_SESSION_NAME
     }
 
-    config_db.set_entry('ACL_RULE',
-            (MIRROR_ACL_TABLE_NAME, MIRROR_ACL_RULE_NAME), acl_rule_info)
+    if v4_table:
+        config_db.set_entry('ACL_RULE',
+                (v4_table, MIRROR_ACL_RULE_NAME), acl_rule_info)
 
     acl_rule_info = {
```

Comment:
I don't think it makes a huge difference but is there any reason these are defined outside of the if-check if they're only used inside of the if-checks?